README.md

```diff
@@ -9,25 +9,25 @@ Web-app to visualize nodes and links on a map for Freifunk open mesh network.
 _Some similar features might have been implemented/merged_
 
 - Map layer modes (Allow to set a default layer based on time combined with a stylesheet)
-- Updates for selected node or list (incl. image stats cache-breaker)
-- Node filter is implemented
+- Automatic updates for selected node or list (incl. image stats cache-breaker)
+- Node filter
 - Zoom level for a node (`nodeZoom`) - Zoom level 22 available, but it is to close for a click
 - Formatted Code
 - Translation support - https://crowdin.com/project/meshviewer - Contact us for new languages
   - Currently available: en, de & fr
 - Grunt inline for some css and js - less requests
 - Icon font with needed icons only
-- Upgrade to Grunt v1.x (Tested with Node.js 4/6 LTS, 7 on Linux, 7 OSX & W**)
-  - Moved css and some js inline
-- Remove bower in favour of npm/yarn
+- Grunt upgraded to v1.x (Tested with Node.js 4/6 LTS, 7 on Linux, 7 OSX & W**)
+  - css and some js moved inline
+- npm/yarn in favour of bower
   - Load only moment.js without languages (Languages are included in translations)
-  - Remove unneeded components (es6-shim, tablesort, numeraljs, leaflet-providers, jshashes)
+  - unneeded components removed (es6-shim, tablesort, numeraljs, leaflet-providers, jshashes)
 - RBush v2 - performance boost in last versions (Positions labels and clients on the map)
-- Remove ruby dependency
+- ruby dependency removed
```

Comment:
Ruby is uppercase on there site. Sass is the same thing (but if you refer to file extension/program it is lowercase). In case lowercase is correct and first word maybe put quotes around